### PR TITLE
update README to revise nav2_bringup guide

### PR DESCRIPTION
**Pull request by yechun1**

### CODE REVIEW

File: nav2_bringup/README.md [hunk 1]
```diff
@@ -17,32 +17,37 @@ Pre-requisites:
 * A Gazebo world for simulating the robot (see Gazebo tutorials)
 * A map of that world saved to a map.pgm and map.yaml (see ROS Navigation tutorials)
 
-Launch Gazebo and Rviz2
+Terminal 1: Launch Gazebo and Rviz2
 
-`ros2 launch nav2_bringup gazebo_rviz2_launch.py world:=<full/path/to/gazebo.world>`
+```
+export GAZEBO_MODEL_PATH=$GAZEBO_MODEL_PATH:<full/path/to/turtlebot3_simulations/turtlebot3_gazebo/models>
```
Comment (mkhansen-intel) [inline nav2_bringup/README.md]: We should either change this to a Turtlebot3 based tutorial or keep this more generic and like </full/path/to/my_robot/gazebo/models>

Comment (mkhansen-intel) [reply nav2_bringup/README.md]: You could also have an example like in the Terminal 2 section below.

Comment (yechun1) [reply nav2_bringup/README.md]: fixed in 9fa61d3

**Comment by yechun1** [comment]
@mkhansen-intel please help review again.

**Comment by mkhansen-intel** [comment]
@yechun1 - I actually haven't tried the map:=occupancy param with the new changes myself. Does that work for you? I can try it in the morning and if it works I'll approve. Sorry for the delay.

**Comment by yechun1** [comment]
Yes, it could work in my local environment.

File: nav2_bringup/README.md [hunk 2]
```diff
@@ -62,7 +68,9 @@ Notes:
 ```
 
 In RVIZ:
-* Localize the robot using “2D Pose Estimate” button.
+* Add "map" to subscribe topic "/occ_grid"
+* Add "RobotModel", set "Description Source" with "File", set "Description File" with "turtlebot3_burger.urdf(from turtlebot3_description package)"
```
Comment (mkhansen-intel) [inline nav2_bringup/README.md]: This should be generic with a turtlebot3 example:

set "Description FIle" with the name of the urdf file for your robot (example: turtlebot3_burger.urdf)

File: nav2_bringup/README.md [hunk 3]
```diff
@@ -62,7 +68,9 @@ Notes:
 ```
 
 In RVIZ:
-* Localize the robot using “2D Pose Estimate” button.
+* Add "map" to subscribe topic "/occ_grid"
```
[the remark this answers is not in the document]
Comment (yechun1) [reply nav2_bringup/README.md]: fixed in 98fd017

File: nav2_bringup/README.md (hunk 2 above)
Comment (yechun1) [reply nav2_bringup/README.md]: fixed in 98fd017.

**Pull request merged**